fairlearn/metrics/_metric_frame.py

```diff
@@ -189,6 +189,32 @@ class MetricFrame:
 
         .. deprecated:: 0.7.0
             `metric` will be removed in version 0.10.0, use `metrics` instead.
+
+    Examples
+    --------
+    >>> from fairlearn.metrics import MetricFrame, selection_rate
+    >>> from fairlearn.datasets import fetch_adult
+    >>> data = fetch_adult(as_frame=True)
```

Comment:
Rather than yet another use of the Adult Census dataset, you could set up a few short arrays like the tests to, with values which are easy to check by inspection.

Reply:
I agree that would be more useful. I will change it tomorrow.